apps/test/util/testUtils.js

```diff
@@ -132,65 +132,6 @@ exports.runOnAppTick = function (app, tick, fn) {
 };
 
 /**
- * Deep equality check of two values with more useful assertion failure
- * message.  Depends on lodash isEqual.
- * @param {*} left
- * @param {*} right
- */
-exports.assertEqual = function (left, right) {
```

Comment:
I actually touched those files too (I think NetSim might be the only client for the commands stuff).